backend/src/Response/CaptainFinancialSystem/CaptainFinancialDuesResponse.php

```diff
@@ -19,4 +19,8 @@ class CaptainFinancialDuesResponse
     public DateTime $startDate;
 
     public DateTime $endDate;
+    
+    public array $paymentsFromCompany;
```

Comment:
When there won't be payments from company, then this field will be null, so that it will cause an error online.